string topicName
float64 newValue
---
string setAnswer
